{{cookiecutter.project_slug}}/config/urls.py

```diff
@@ -35,6 +36,10 @@
     path("api/", include("config.api_router")),
     # DRF auth token
     path("auth-token/", obtain_auth_token),
+    path("api/schema/", SpectacularAPIView.as_view(), name="schema"),
+    path(
+        "swagger/", SpectacularSwaggerView.as_view(url_name="schema"), name="swagger-ui"
```

Comment:
If I'm not mistaken, swagger is the name of the underlying library generating the docs, and DRF spectacular offers another option. IMO it's an implementation detail that shouldn't leak in the URL path. How about `api/docs/` instead? The URL name could be changed as well, but that's less important.

Reply:
Fixed this